claims_hosp/delphi_claims_hosp/modify_claims_drops.py

```diff
@@ -57,5 +57,5 @@ def modify_and_write(data_path, logger, test_mode=False):
             dfs_list.append(dfs)
         else:
             dfs.to_csv(out_path, index=False)
-            logger.info(f"Wrote {out_path}")
+            logger.info("Wrote modified csv", output_path=out_path)
```

Comment:
```suggestion
            logger.info("Wrote modified csv", filename=out_path)
```
I would go back to calling this `filename` since it is an actual file name and not just a path, plus you call it filename in [doctor_visits/delphi_doctor_visits/modify_claims_drops.py](https://github.com/cmu-delphi/covidcast-indicators/pull/2029/files#diff-69d81b3cc6c9eb738d7c02706a6ae96da94103c158a1572b55639c7c9bafa098)